src/main/java/frc/robot/subsystems/Tabs/TabDriveBase.java

```diff
@@ -22,19 +22,19 @@
     // DriveBaseSubsystem object
     private DriveBaseSubsystem mDriveBaseSubsystem;
 
-    private AHRS mGyro;
+    //private AHRS mGyro;
```

Comment:
Assume these were all removed for testing? If so, uncomment before merging this into main